### DISCUSSION TITLE
[Import] Problem migrating from instapaper

### DISCUSSION THREAD

**Discussion by Gludred** [wallabag/wallabag#6457] [Q&A]
hello community. I have a problem migrating my instapaper data. I get an error message: "This form should not contain extra fields.". I checked the CSV file. I don't understand why this isn't working. Thank you for your help

**Comment by j0k3r** [comment]
Are you on PHP 8.1?
If so it could be the same as this issue: https://github.com/wallabag/wallabag/issues/5631

**Reply by Gludred** [reply]
I'm currently on PHP 8.0.23 (Synology DS920+ DSM 7.1)

I have already viewed this issue.

Thks

**Reply by j0k3r** [reply]
Which version of wallabag are you using?

**Reply by Gludred** [reply]
Wallabag 2.5.4

**Reply by j0k3r** [reply]
Any chance to try on PHP 7.4 or 8.2 ?

**Reply by Gludred** [reply]
I can try
